config/config.go

```diff
@@ -1271,22 +1270,8 @@ func (cfg *InstrumentationConfig) ValidateBasic() error {
 	if cfg.PyroscopeTrace && cfg.PyroscopeURL == "" {
 		return errors.New("pyroscope_trace can't be enabled if profiling is disabled")
 	}
-	// if there is not InfluxURL configured, then we do not need to validate the rest
-	// of the config because we are not connecting.
-	if cfg.InfluxURL == "" {
-		return nil
-	}
-	if cfg.InfluxToken == "" {
-		return fmt.Errorf("token is required")
-	}
-	if cfg.InfluxOrg == "" {
-		return fmt.Errorf("org is required")
-	}
-	if cfg.InfluxBucket == "" {
-		return fmt.Errorf("bucket is required")
-	}
-	if cfg.InfluxBatchSize <= 0 {
-		return fmt.Errorf("batch size must be greater than 0")
+	if cfg.TraceBufferSize < 0 {
+		return fmt.Errorf("trace buffer size must be greater than 0")
 	}
```

Comment:
Can it be 0? Do we not want `<=`

Reply:
technically speaking, 0 could work although ofc that's a bad idea. The reasoning for switching here was so that a value of 0 won't fail for when the default value isn't used in some tests, and that I wasn't sure where to stop. While 0 is not reccomended, neither is 1. combining those two things resulted in just switching this to 0